homework2/sass/common.scss

```diff
@@ -0,0 +1,54 @@
+
+
+$color-secondary-text: #7F7F7F;
```

Comment:
variables and mixins should split into different files

Reply:
Done